docs/using/recipe_code/oracledb.dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jupyter/base-notebook
+
+## Install java, javac and alien
+USER root
+RUN apt-get update --yes && \
+    apt-get install --yes --no-install-recommends software-properties-common && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*  && \
+    apt-get update --yes && \
+    add-apt-repository universe && \
+    apt-get update --yes && \
+    apt-get install --yes --no-install-recommends alien default-jre default-jdk openjdk-11-jdk libaio1 && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+## Then install Oracle SQL Instant client, SQL+Plus, tools and JDBC.
+## Note: You may need to change the URL to a newer version.
+## See: https://www.oracle.com/es/database/technologies/instant-client/linux-x86-64-downloads.html
+RUN mkdir "/opt/oracle"
+WORKDIR "/opt/oracle"
+RUN wget --progress=dot:giga https://download.oracle.com/otn_software/linux/instantclient/2111000/oracle-instantclient-basiclite-21.11.0.0.0-1.el8.x86_64.rpm && \
+    alien -d --scripts oracle-instantclient-basiclite-21.11.0.0.0-1.el8.x86_64.rpm && \
+    dpkg -i oracle-instantclient-basiclite_21.11.0.0.0-2_amd64.deb && \
+    wget --progress=dot:giga https://download.oracle.com/otn_software/linux/instantclient/2111000/oracle-instantclient-sqlplus-21.11.0.0.0-1.el8.x86_64.rpm && \
+    alien -d  --scripts oracle-instantclient-sqlplus-21.11.0.0.0-1.el8.x86_64.rpm && \
+    dpkg -i oracle-instantclient-sqlplus_21.11.0.0.0-2_amd64.deb && \
+    wget --progress=dot:giga https://download.oracle.com/otn_software/linux/instantclient/2111000/oracle-instantclient-tools-21.11.0.0.0-1.el8.x86_64.rpm && \
+    alien -d --scripts oracle-instantclient-tools-21.11.0.0.0-1.el8.x86_64.rpm && \
+    dpkg -i oracle-instantclient-tools_21.11.0.0.0-2_amd64.deb && \
+    wget --progress=dot:giga https://download.oracle.com/otn_software/linux/instantclient/2111000/oracle-instantclient-jdbc-21.11.0.0.0-1.el8.x86_64.rpm && \
+    alien -d --scripts oracle-instantclient-jdbc-21.11.0.0.0-1.el8.x86_64.rpm && \
+    dpkg -i oracle-instantclient-jdbc_21.11.0.0.0-2_amd64.deb
+## Remove temporary files to avoid issues with `fix-permissions` commands
+RUN rm "*.deb" && rm "*.rpm" & chown -R "${NB_UID}":"${NB_GID}" "${HOME}/.rpmdb"
```

Comment:
It's better to have this rm in a previous command, because it will make image size significantly smaller.

Reply:
Hi, @mathbunnyru 

Thanks for the command but I am not sure If I understand what do you mean.

Could you be more specific?

Thanks

Reply:
I added a commit suggestion.
When you download some file in one RUN command and then remove it in the next RUN command, the result image size will include the size of this file, because docker saves all the layers.
That's why a good practice is to download something, use it and then remove it in one RUN command.